docs/src/user-guide/install.md

```diff
@@ -1,40 +1,111 @@
+If you already have Rust installed, make sure you're using the latest version by running:
+```bash
+rustup update
 ```
+Then, install the remaining dependencies.
 
-Last, to build and install anoma you can run the following command:
+**Ubuntu:** running the following command should install everything needed:
+```bash
+sudo apt-get install -y make git-core libssl-dev pkg-config libclang-12-dev
+```
+**Mac:** installing the Xcode command line tools should provide you with almost everything you need:
+```bash
+xcode-select --install
+```
+Now, that you have all dependencies installed you can clone the source code from the [Anoma repository](https://github.com/anoma/anoma) and build it with:
 
 ```bash
+git clone https://github.com/anoma/anoma.git --single-branch --branch $ANOMA_TESTNET_BRANCH
+cd anoma 
 make install
 ```
 
-## Nix
+Once done, you can go to [Setting up Namada](../testnets/internal-testnet-1/#setting-up-namada) section.
+
+### From Binaries
+
+>⚠️ During internal and private testnets, links to releases provided in this section might be outdated and inconsistent to the current testnet.
+>To avoid any issues, we recommend you [build from source](#build-from-source).
+
+If you'd like to install Namada from binaries you will have to install some dependencies first: [Rust](https://www.rust-lang.org/tools/install), [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Tendermint](https://docs.tendermint.com/master/introduction/install.html) `0.34.x` and GLIBC `v2.29` or higher.
+
+First install Rust.
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+At the end of the installation, make sure that Cargo's bin directory ($HOME/.cargo/bin) is available on your PATH environment variable. You can either restart your shell or run `source $HOME/.cargo/env` to continue.
+
+If you already have Rust installed, make sure you're using the latest version by running:
+```bash
+rustup update
+```
+
+Then, install Git. 
+
+**Ubuntu:** running the following command should install Git:
+```bash
+sudo apt-get install git
+```
+**Mac:** installing the Xcode command line tools should provide you with almost everything you need:
+```bash
+xcode-select --install
+```
+Next, let's install Tendermint. 
+
+You can either follow the instructions on the [Tendermint guide](https://docs.tendermint.com/master/introduction/install.html) or download the `get_tendermint.sh` script from the [Anoma repository](https://github.com/anoma/anoma/blob/master/scripts/install/get_tendermint.sh) and execute it (will ask you for `root` access):
+```bash
+curl -LO https://raw.githubusercontent.com/anoma/anoma/master/scripts/install/get_tendermint.sh
+chmod +x get_tendermint.sh
+./get_tendermint.sh
+```
+Now, that you have all dependencies installed you can download the latest binary `v0.5.0` from our [releases](https://github.com/anoma/anoma/releases) page by replacing `{platform}` by either `Linux-x86_64` or `Darwin-x86_64` for Mac:
```

Comment:
no `Darwin-x86_64` prebuilt binaries for v0.5.0 🥲 

It might be enough to just link to https://github.com/anoma/anoma/releases and indicate the reader to download the latest .tar.gz for their OS/architecture, if that isn't present then they'll have to build from source